### PR TITLE
Feature/implements dark theme

### PR DESCRIPTION
**Pull request by comountainclimber**
implements the dark theme 😎 

https://github.com/CityOfZion/neon-wallet/issues/1239

### CODE REVIEW

File: app/components/AmountsPanel/AmountsInfoBox/AmountsInfoBox.scss [hunk 1]
```diff
@@ -7,9 +7,9 @@
   padding: 0 12px;
   box-sizing: border-box;
   font-size: 14px;
-  color: #5c677f;
+  // color: #5c677f;
```
Comment (evgenyboxer) [inline app/components/AmountsPanel/AmountsInfoBox/AmountsInfoBox.scss]: is there a need for this comment?

File: app/containers/App/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -16,110 +16,117 @@ import TokenSaleIcon from '../../../assets/navigation/tokens.svg'
...
 import { ROUTES } from '../../../core/constants'
 
-import LogoWithoutText from '../../../assets/images/logo-without-text-black.png'
+import LightLogoWithoutText from '../../../assets/images/logo-without-text-black.png'
+import DarkLogoWithoutText from '../../../assets/images/logo-without-text.png'
 
 type Props = {
-  className: string
+  className: string,
+  theme: 'Light' | 'Dark'
```
Comment (evgenyboxer) [inline app/containers/App/Sidebar/Sidebar.jsx]: I think its best if the theme prop will have a `ThemeType` (this type can be exported from where the theme was defined)

```
const THEME = {
 LIGHT: 'Light',
 DARK: 'Dark'
}

type ThemeType = 'Light' | 'Dark'
```

File: app/containers/Settings/Settings.jsx [hunk 3]
```diff
@@ -257,18 +258,19 @@ export default class Settings extends Component<Props, State> {
...
                 <StyledReactSelect
                   settingsSelect
-                  isDisabled
+                  onChange={this.updateThemeSettings}
+                  isSearchable={false}
                   transparent
+                  options={[
+                    { value: 'Light', label: 'Light' },
+                    { value: 'Dark', label: 'Dark' }
+                  ]}
```
Comment (zsiglin) [inline app/containers/Settings/Settings.jsx]: For consistency's sake should these be mapped constants from `core/constants` like currency and explorer options are?

**Comment by zsiglin** [comment]
Looks like you need to update snapshots to get the tests passing. But the dark theme looks great. Glad this could be carried across the finish line. 👍 👍

**Comment by comountainclimber** [comment]
Thanks for taking a look @evgenyboxer and @zsiglin it is much appreciated 👍 let me know if you have any other concerns

**Pull request merged**